open_academy/views/course_form.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_course_form"  model="ir.ui.view">
```

Comment:
Please follow this documentation to name your views https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action